appveyor.yml

```diff
@@ -49,9 +49,9 @@ install:
     - cmd: set PATH=%CONDA_INSTALL_LOCN%;%CONDA_INSTALL_LOCN%\scripts;%PATH%
     - cmd: set PYTHONUNBUFFERED=1
 
-    - cmd: conda config --add channels conda-forge
     - cmd: conda config --set show_channel_urls true
     - cmd: conda install -c pelson/channel/development --yes --quiet obvious-ci
+    - cmd: conda config --add channels conda-forge
```

Comment:
This is just reverting a workaround for this issue ( https://github.com/Anaconda-Platform/support/issues/45 ), which was fixed by this PR ( https://github.com/conda/conda/pull/2670 ) and appeared in `conda` 4.1.1.